src/app.js

```diff
@@ -14,10 +14,22 @@ export const App = () => (
   <Provider store={store}>
     <h1>Hello, I`m App Component!</h1>
     <Counter />
-    <Icon icon="Plus" size={32} />
-    <Icon icon="Edit" viewBox="0 0 50 50" />
-    <Icon icon="Plus" size={32} className="icon" color="#FFFFFF" />
     <Icon icon="Plus" size={32} className="icon" color="#FFB800" />
     <Icon icon="Edit" viewBox="0 0 50 50" color="#FFB800" />
+    <Icon icon="Back" color="#FFB800" />
+    <Icon icon="Calendar" color="#FFB800" />
+    <Icon icon="CheckboxFalse" color="#FFB800" />
+    <Icon icon="CheckboxNull" color="#FFB800" />
+    <Icon icon="CheckboxTrue" color="#FFB800" />
+    <Icon icon="CheckFalse" color="#FFB800" />
+    <Icon icon="CheckNull" color="#FFB800" />
+    <Icon icon="CheckTrue" color="#FFB800" />
+    <Icon icon="CloseButton" color="#FFB800" />
+    <Icon icon="DropDown" color="#FFB800" />
+    <Icon icon="DropDownButton" viewBox="0 0 50 50" color="#FFB800" />
+    <Icon icon="LogOut" viewBox="0 0 50 50" size={100} />
+    <Icon icon="Menu" viewBox="0 0 50 50" size={100} color="#FFB800" />
+    <Icon icon="Profile" viewBox="0 0 50 50" size={100} color="#FFB800" />
+    <Icon icon="Search" viewBox="0 0 50 50" size={100} color="#FFB800" />
```

Comment:
Why we provide all icons?

Reply:
We had problems with viewBox, that for some icons default viewBox is normal (for most of them, and for another small, and icons was looking cut, so that’s why I’ve provided them all, and also for checking that we have added all that we will need. Should I delete them?

Reply:
I`ve changed it